src/dymaptic.GeoBlazor.Core/Scripts/definitions.d.ts

```diff
@@ -106,4 +106,21 @@ export interface DotNetLayerView {
     suspended: boolean;
     updating: boolean;
     visible: boolean;
+}
+export interface DotNetBaseListItem {
```

Comment:
These two interfaces should still be deleted.

Reply:
removed